Error when can't bind for ident server, but doesn't exits

```
2022-03-11 06:15:55 [INFO] The Lounge v4.3.1-rc.1 (Node.js 14.19.0 on linux x64)
2022-03-11 06:15:55 [INFO] Configuration file: /etc/thelounge/config.js
2022-03-11 06:15:55 [INFO] Available at http://127.0.0.1:9000/ in private mode
2022-03-11 06:15:55 [INFO] Package thelounge-theme-solarized v1.1.9 loaded
2022-03-11 06:15:55 [ERROR] Identd server error: Error: listen EADDRNOTAVAIL: address not available [IPv6 address]:1113
```

The server errors when it tries to bind for the identd but is unable to. However, it doesn't exits as you'd expect it to, and won't continue to recover or do anything, like load users. It just gets stuck there and won't do anything else.

It happily listened to a SIGTERM, though:
```
2022-03-11 06:17:13 [INFO] Exiting...
2022-03-11 06:17:13 [INFO] Clearing prefetch storage folder, this might take a while...
```

--- a/src/identification.js
+++ b/src/identification.js
@@ -27,7 +27,9 @@ class Identification {
 
 			const server = net.createServer(this.serverConnection.bind(this));
 
-			server.on("error", (err) => log.error(`Identd server error: ${err}`));
+			server.on("error", (err) => {
+				startedCallback(this, err);
+			});
 
 			server.listen(
 				{

--- a/src/server.js
+++ b/src/server.js
@@ -199,7 +199,12 @@ module.exports = function (options = {}) {
 			Helper.config.themeColor = defaultTheme.themeColor;
 		}
 
-		new Identification((identHandler) => {
+		new Identification((identHandler, err) => {
+			if (err) {
+				log.error(`Could not start identd server, ${err.message}`);
+				process.exit(1);
+			}
+
 			manager.init(identHandler, sockets);
 		});
 

